conans/client/graph/python_requires.py

```diff
@@ -34,7 +34,13 @@ def __call__(self, require):
             result = self._proxy.get_recipe(r, False, False, remote_name=None,
                                             recorder=ActionRecorder())
             path, _, _, reference = result
-            module, _ = parse_conanfile(path)
+            module, filename = parse_conanfile(path)
+
+            # Check for alias
+            conanfile = parse_module(module, filename)
```

Comment:
I think I'll prefer to really have an injected loader object into PythonRequires (yes, runtime cycle between objects).
We could make ``ConanFileLoader`` be the one that really instantiates ``PythonRequires`` object, passing a referent to itself to it.

This can't go to 1.9.2, a bit risky in my opinion.